modules/tinymce/src/core/test/ts/browser/commands/LineHeightTest.ts

```diff
@@ -0,0 +1,64 @@
+import { Log, Pipeline, Step } from '@ephox/agar';
+import { Assert, UnitTest } from '@ephox/bedrock-client';
+import { TinyApis, TinyLoader } from '@ephox/mcagar';
+import Editor from 'tinymce/core/api/Editor';
+
+UnitTest.asyncTest('browser.tinymce.core.commands.LineHeightTest', (success, failure) => {
+  TinyLoader.setupLight((editor: Editor, success, failure) => {
+    const api = TinyApis(editor);
+
+    const sAssertHeight = (value: string) => Step.sync(() => {
+      const current = editor.queryCommandValue('LineHeight');
```

Comment:
It might be better to not expose this as api but instead just query the dom in the test to assert the state. Unless some client for some reason explicitly requires this.

Reply:
It was part of the spec on the Jira ticket for parity with similar features. It's also used inside the UI.